src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -21,6 +21,11 @@ public static partial class Kernel32
         /// </summary>
         public const int PIPE_UNLIMITED_INSTANCES = 255;
 
+        /// <summary>
+        /// All processes start at this shutdown level
+        /// </summary>
+        public const int DEFAULT_SHUTDOWN_LEVEL = 0x280;
```

Comment:
Where is this constant defined and documented? I can't find it on MSDN or in common .h files.